sdk/test-utils/perfstress/src/tests.ts

```diff
@@ -41,14 +41,14 @@ export abstract class PerfStressTest<TOptions = {}> {
   }
 
   // Before and after running a bunch of the same test.
-  public globalSetup?(): void | Promise<void>;
-  public globalCleanup?(): void | Promise<void>;
+  public globalSetup(): void | Promise<void> {}
+  public globalCleanup(): void | Promise<void> {}
 
-  public setup?(): void | Promise<void>;
-  public cleanup?(): void | Promise<void>;
+  public setup(): void | Promise<void> {}
+  public cleanup(): void | Promise<void> {}
 
-  public run?(abortSignal?: AbortSignalLike): void;
```

Comment:
Question: Should we remove this as we only ever offer async methods for our APIs?

Reply:
In a future PR, as it would involve modifying the framework.

Reply:
Yes, I would remove all the sync code in a separate PR.